apps/condo/domains/user/schema/ForgotPasswordService.js

```diff
@@ -75,7 +75,9 @@ const ForgotPasswordService = new GQLCustomSchema('ForgotPasswordService', {
                 const users = await User.getAll(context, { phone, type: STAFF })
 
                 if (isEmpty(users)) {
-                    throw new Error(`${WRONG_PHONE_ERROR}] Unable to find user when trying to start password recovery`)
+                    // We don't need to say to client whether we have users with this phone number
+                    // because it is not safety.
+                    return { status: 'ok' }
```

Comment:
Throwing an exception interrupts execution flow and `return` will never be reached.